spec/features/board_pages/board_pages_spec.rb

```diff
@@ -38,6 +38,28 @@
         expect(page).to have_text board.name
         expect(page).not_to have_text node_board.name
       end
+
+      context 'creating a board using a template', js: true do
+        it 'creates the board using the template data' do
+          template_path = Rails.root.join('spec/fixtures/files/methodologies/')
+          allow(Methodology).to receive(:pwd).and_return(template_path)
+
+          board
+          visit project_boards_path(current_project)
+
+          click_link 'Create new methodology...'
+
+          find('#modal-board-new', visible: true)
+          find('#template').find(:option, 'Methodology Template v3').select_option
+
+          click_button 'Add methodology'
```

Comment:
```suggestion
          expect {
            click_button 'Add methodology'
          }.to change(Board, :count).by(1)
```
Aaron, I think changing the expectation to assert a board is _created_ is more concrete than just checking for the attributes of the board.